Java/Algorithms/Moore_Algorithm.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Scanner;
+public class MoreAlgo {
+	public static void check(int arr[]) {
+		int count=0;
+		for(int i=0;i<arr.length-1;i++) {
+			count=1;
+			for(int j=i+1;j<arr.length;j++) {
+				if(arr[j]==arr[i]) {              // Counting the number by matching it with
+					count++;  					 // other numbers.
+				}
+			}
+			if(count>arr.length/2) {
+				System.out.println("Number Occuring More than 'n/2'="+arr[i]);
+				System.exit(0);
+			}
+		}
+		System.out.print("NO majority Elements ");
+	}
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Enter Number of elements (Eg. 6) : ");
```

Comment:
Don't use shorthands. We cannot expect everyone to understand them.

Reply:
Okay , I will make the changes

Reply:
@HarshCasper I have updated the file ,please let me know if any changes are required.